### PR TITLE
Add Sol.app v1.1.20

### PR DESCRIPTION
**Pull request by g-cqd**
Creation of a cask for Sol app, an Open Source macOS launcher by @ospfranco.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/sol.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "sol" do
+  version "1.1.20"
+  sha256 :no_check
+
+  url "https://github.com/ospfranco/sol/releases/latest/download/Sol.zip"
```
Comment (SMillerDev) [inline Casks/sol.rb:5]: Please use the download for the specific version

File: Casks/sol.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  name "Sol"
+  desc "Creative Open Source Launcher"
```
Comment (SMillerDev) [inline Casks/sol.rb:7]: I think this captures the usage more
```suggestion
  desc "Launcher & command palette"
```

File: Casks/sol.rb (hunk 1 above)
Comment (aemi-dev) [reply Casks/sol.rb]: I changed to the version URL. But the repo doesn't support the livecheck at the moment so I skipped it.

Comment (aemi-dev) [reply Casks/sol.rb]: The livecheck retrieve the latest tag whereas it should retrieve the latest release in my tests.
Screenshot 2022-05-30 at 14 06 21

File: Casks/sol.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  sha256 "e7880031cec94a31825aaaac994bdf8e5711763c35e50938fc2c8b3d9868be98"
+
+  url "https://github.com/ospfranco/sol/releases/download/#{version}/Sol.zip"
+  name "Sol"
+  desc "Launcher & command palette"
+  homepage "https://github.com/ospfranco/sol"
+
+  livecheck do
+    skip "Sol does not currently support livecheck."
+  end
```
Comment (SMillerDev) [inline Casks/sol.rb:12]:
```suggestion
  livecheck do
    url :url
    strategy :github_latest
  end
```

Comment (aemi-dev) [reply Casks/sol.rb]: I missed the line in the documentation that exactly expressed that... I'm sorry for wasting your time 😅

**Review by SMillerDev — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @aemi-dev

**Comment by aemi-dev** [comment]
> Thank you @aemi-dev 

Thank you too 🙏

**Pull request merged**